models/models.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class meeting__scheduler(models.Model):
+    _name = 'meeting__scheduler.meeting__scheduler'
+    _description = 'meeting__scheduler.meeting__scheduler'
+
+    name = fields.Char()
```

Comment:
@NilsNeeb Always set a label to fields: fields.Char(string="set label like this") then those can be translated later on. You can actually use german for the labels